src/docs/development/platform-integration/web.md

```diff
@@ -9,32 +9,36 @@ As of the 1.12 release, web support for Flutter
 is available on the beta channel.
 **We don't recommend deploying a web app to production.**
 
+### What sort of apps are not ideal for Flutter on the web?
```

Comment:
It may be better to have the question be more positive : What scenarios are ideal for Flutter on the web?